mono/mini/mini-native-types.c

```diff
@@ -42,7 +42,6 @@ typedef struct {
 	short compare_op;
 } MagicTypeInfo;
 
-
```

Comment:
unrelated whitespace change

Reply:
That's already in another approved PR, I can remove it here.
It would be nice if we could actually cleanup the code formatting.